code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -484,13 +487,50 @@
 	moduleselect_icon = "service"
 	special_light_key = "service"
 	hat_offset = 0
+	whitelisted_reagents += /datum/reagent/consumable/orangejuice //Add here anything it can synthetize regardless of emag status

```

Comment:
This should be a list of the reagent ids.

Reply:
You mean like only the orangejuice part?

Reply:
```suggestion
	var/list/whitelisted_reagents = list("orangejuice", "tomatojuice")
```

Reply:
Also, how to make it so it gets added to the existing list? Is this the right way?

Reply:
For example, lets say "orangejuice" is already inside the list on line 37, but I want to add "tomatojuice" just to the butler borg.
This way both the medical and the butler could make orangejuice without me needing to add on both a repeated list.

Reply:
When you want to add to it:
>whitelisted_reagents += "heparin"

or
>whitelisted_reagents += list("polonium", "sodiumthiopental")

Reply:
The fuck-up was that on line 37 I had defined the variable wrong. Now I think this part will work.

Reply:
It didnt work this way. Changed to whitelisted_reagents.Add() and now Travis do not give me any error.